packages/dev/docs/pages/releases/2022-07-09.mdx

```diff
@@ -0,0 +1,274 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: In this release, we focused on improvements to bundle size and developer experience. These include tree shaking improvements, large dependency removal, API simplification, TypeScript improvements, and a big documentation overhaul. And as always, we also have lots of bug fixes for our existing components.
+date: 2022-07-29
+---
+
+# July 29, 2022 Release
+
+In this release, we focused on improvements to bundle size and developer experience. These include tree shaking improvements, large dependency removal, API simplification, TypeScript improvements, and a big documentation overhaul. And as always, we also have lots of bug fixes for our existing components.
+
+## Developer experience
+
+Based on feedback from the community, we have been working on a number of developer experience improvements for React Aria. We have updated many of our older hooks to follow our newer API conventions which reduce boilerplate and details you need to worry about. We have added states such as `isPressed`, `isSelected`, and `isDisabled` to the return values of many hooks, which can be used for styling purposes. We have also reduced the number of hooks you need to call, and the number of arguments you need to manually pass between them.
+
+We have also made many improvements to the React Aria documentation. We’ve added styled examples in CodeSandbox for many of our hooks, which demonstrate various styling libraries and techniques such as Tailwind CSS, Styled Components, CSS modules, and more. We have also added a “Usage” section to all of our docs, which includes various examples showing the features and options of components built with our hooks. Finally, all React Aria examples now use the `react-aria` and `react-stately` mono packages rather than individual component packages, which reduces the number of npm packages you need to install and manage.
+
+We have also improved the TypeScript experience in this release. All of our packages including both individual packages and mono packages (e.g. `@adobe/react-spectrum`, `react-aria`, and `react-stately`) now export all of their types. This allows you to use them in your own components, without installing extra packages. In addition, all of our return types now support both HTML and SVG elements.
```

Comment:
Mega nit: commas for clarity, thought for a sec that this meant all our packages include both the individual and monopackages
```suggestion
We have also improved the TypeScript experience in this release. All of our packages, including both individual packages and mono packages (e.g. `@adobe/react-spectrum`, `react-aria`, and `react-stately`), now export all of their types. This allows you to use them in your own components, without installing extra packages. In addition, all of our return types now support both HTML and SVG elements.
```